examples/Catena4430_Sensor/Catena4430_cMeasurementLoop_SDcard.cpp

```diff
@@ -571,4 +575,75 @@ cMeasurementLoop::updateFromSd(
         }
     }
 
+void
+cMeasurementLoop::handleSdTTNv3Migrate(
+    void
+    )
+    {
+    static const char * const sMigrate = "MIGRATE.V3";
```

Comment:
This should be a constexpr at the top of the file, or in `...params.h` (if this app has such a header file). The variable should be `gkMigrateFileName` to be more descriptive so it would be:

```c++
constexpr char gkMigrateFileName[] = "MIGRATE.V3";
```

Reply:
declared as global in the top of same file